### PR TITLE
Dynamodb

### PR DESCRIPTION
**Pull request by elvia-si**
Please check my code. Terraform apply is working locally. Thanks

### CODE REVIEW

**Review by Shane-Frost — APPROVED** [review]
a - ok

**Review by Shane-Frost — APPROVED** [review]
a - ok

File: elasticsearchEC2.tf [hunk 1]
```diff
@@ -0,0 +1,85 @@
+module "network" {
+  source             = "./vpc"
+  public_name               = "my_vpc"
```
Comment (ash-ley) [inline elasticsearchEC2.tf]: This refers to the public subnet name in the VPC which should be Kabana-Public-A

**Review by ash-ley — APPROVED** [review]

**Review by himatvarsani-cloudreach — APPROVED** [review]

File: elasticsearchEC2.tf (hunk 1 above)
Comment (elvia-si) [reply elasticsearchEC2.tf]: I'm going to change that. Thanks

**Pull request closed without merging**